src/test/java/spoon/support/compiler/classpath/ComputeClasspathTest.java

```diff
@@ -48,32 +47,8 @@ public SpoonCompiler createCompiler(Factory factory) {
 	}
 
 	@Test
-	public void testTemplateClasspath()
```

Comment:
Check old test cases: they tested `computeTemplateClasspath` and `computeJdtClassPath`. These methods were privates.